app/fn-guru/src/main/frontend/src/pages/app/exec-list/components/row-actions.tsx

```diff
@@ -28,7 +36,28 @@ export default function ({row}: Props) {
                 <DropdownMenuItem onClick={() => {
                     navigate(`/executions/${row.original.id}`)
                 }}>View</DropdownMenuItem>
+                {trigger && <TriggerSwitch triggerId={trigger.id} status={trigger.status}/>}
             </DropdownMenuContent>
         </DropdownMenu>
     )
 }
+
+
+type TriggerSwitchProps = { triggerId: string, status: string }
+const TriggerSwitch: FC<TriggerSwitchProps> = ({triggerId, status}) => {
+    const [setTriggerStatus, setTriggerRequested, loading, error] = useSetTriggerStatus()
+    const current = status === "Active"
```

Comment:
current is a pretty bad name for a status variable - and you don't really leverage it.

Maybe do something like

```typescript
type TriggerStatus = 'Active' | 'Inactive'

```


```typescript
        <DropdownMenuItem onClick={handleClick}>
            {status == 'Inactive' ? "Activate" : "Deactivate" }
        </DropdownMenuItem>
```